### ISSUE TITLE
Reinstating/New Application for "More" prompt

### ISSUE THREAD

**Issue by JoshJetto** [gryphonshafer/Quizzing-Rule-Book#220]
Proposed Rule: On an INT, MA, or Reference question, if all the information in the answer text has been given but the quizzer is
 missing information in the question text, the quizmaster will say "More".

I would like to propose that we reinstate the "More" prompt for INT and MA questions and along with that expand its application to include CR and CVR question types. Here's why:
1) On INT/MA questions when quizzers jump before the question is completed, they sometimes cut off a quizmaster mid-word and then give their answer from the word that follows the cut-off word, giving all the information required in the question and answer except for the word that was not completed by the quizmaster. For the quizmaster to simply stare at the question silently while the quizzer puzzles over what they might be missing and then to have the time run out and have the quizmaster say, "I never completed word X, and you never said that whole word in your answer, so I have to call you incorrect" is not only unhelpful and discouraging to the quizzers but also feels ridiculous to the observer. The "more" prompt communicates to the quizzer, you have given the correct answer, but there are some words in the question that weren't finished that you need to say, and it directs the quizzer helpfully to requote from the start of the question to fill in what was missing. I think we should bring this part of the "More" prompt back to the rulebook.
2) On CR and CVR questions, one of the major stumbling block issues occurs on so-called 50/50 questions (okay, mainly with CVR questions), in which it is impossible for a quizzer who jumps on the verse reference to determine with certainty the question, and they are left to make a 50/50 guess and hope they picked the right one. An example from this year's material in Luke is found in Luke 2:13 "Suddenly a great company of the heavenly host appeared with the angel, praising God and saying..." Two possible CVR questions here are "Great what?" and "What company." If the "More" prompt were reinstated and expanded to included reference type questions, then a quizzer could jump on Luke 2:13 and give the answer "Great...of the heavenly host" omitting the word "company" from their answer. If the question were "Great what?" the quizmaster would say nothing, because the answer had not been given yet, and so the quizmaster could not yet ask "what is your question" and wouldn't say "more" because "company" is not part of the question text. Hearing nothing the quizzer would then say, "Company," be prompted for the question and say, "Great what?" and be called correct. If the quizzer gave the same answer above (again omitting the word "company"), and then received the prompt "more" from the quizmaster, they would know that the question must be "What company" since they had given everything needed to answer the question but hadn't included the word from the question that was needed. A simpler example given in the rulebook is "Lawless what?" and "What deeds?" A quizzer could say "Lawless" (pause), hear no prompt, then say "Lawless deeds," and know that the question had to be "Lawless what?" Or they could say "Lawless" (pause), hear the "more" prompt, then say "Lawless deeds," and know that the question had to be "What deeds?" The beauty of reinstating and expanding the "more" prompt to include reference questions is that it would, for the expert reference quizzers, eliminate the 50/50 issue. 
For those who might be for the application of the prompt for References but be against the application for INTs and MAs, I would say that the reinstatement provides consistency across the question types and helps quizzers to understand better on Reference questions what the "more" prompt means because they will have heard it used somewhat regularly with the more common question types.

**Comment by jttower** [comment]
I am very much against reinstating the "More" prompt as we discussed this A LOT before we voted to get rid of it.  I'm sure you could go back to those discussions to see the reasoning, but I think the shortest answer is that the "more" prompt was confusing and was not uniformly used, and that we were trying to streamline the QM prompts.  

I understand your arguments above, but again, I feel that we have discussed all sides of this issue.  Regarding INT/MA, if the quizzer has not provided all the information in the question and answer, then they should keep talking, going back to requote the whole verse if necessary.  This also applies to Quote questions in which the QM doesn't get the whole number out, but quizzers can easily be trained to repeat the whole reference before they answer.  Regarding the REF, it is a quizzer's choice to pre-jump.  Regardless of the question type, there is a risk in pre-jumping because you don't have the whole question. Adding in the "more" prompt again does not necessarily remove all confusion on the part of quizzers, in fact, I have seen it ADD confusion instead of clear things up.  If the QM doesn't say "incorrect", then keep talking.  If the QM doesn't say "correct", then keep talking.  This is the easiest way to deal with all of these issues.  

If districts want to use the "more" prompt (or any other prompt for that matter), then they are free to do so, but I don't think we should go back on our decision.  Changing rules back and forth is confusing.

**Comment by 24RMiller** [comment]
I agree with @jttower for the most part in that this just adds confusion. In general I am in favor of less prompts, not more. Quizzers should not need prompted to give more information on ints and mas, and if they follow the good quizzing habit of repeating what the qm says, it shouldn't matter anyways. If districts want to say things like "more" in jrs or B to prompt quizzers to keep talking I have no problem with that, but international level quizzers should know to keep talking.

Regarding CVRs, I do think the 50/50 rule is unfortunate and a change could be made. However, CVRs already require a unique skill and this rule would add even more technicality in the way verses are quoted, making them even more of an "elite" question type. While I would like to see something changed in the way CVRs are given/ruled on, I don't think this is a good solution.

**Comment by l-koral** [comment]
I personally don't agree with the reinstatement of the "more" prompt because it will create more confusion. Especially on CVRs, which are already challenging as it is. (Personally, I think CVRs are too difficult for those without higher-level quizzing skills, but this is another conversation for some other time.)

**Comment by kclimenhaga** [comment]
My practice in district level quizzing has been to use the "Again" prompt on all question types, not just Q/F type questions. I agree that when quizzers have missed part of the question, staring at them is unhelpful, but rather than adding a confusing prompt back in, I would rather expand the use of "Again" to any question where the quizzer has made an error, including missing information.
